src/scripts/OSFramework/OSUI/Pattern/FlipContent/FlipContent.ts

```diff
@@ -72,7 +72,7 @@ namespace OSFramework.OSUI.Patterns.FlipContent {
 		// Triggers the toggle event on the platform
 		private _triggerPlatformEvent(): void {
 			if (this._platformEventOnToggle) {
```

Comment:
Maybe we can just bring that validation to inside the method `triggerPlatformEventplatformCallback`?